kentik_api_library/setup.py

```diff
@@ -102,15 +133,13 @@ def run(self):
     url="https://github.com/kentik/community_sdk_python/tree/main/kentik_api_library",
     license="Apache-2.0",
     include_package_data=True,
+    python_requires=">=3.8, <4",
     install_requires=["dacite>=1.6.0", "requests[socks]>=2.25.0", "typing-extensions>=3.7.4.3", "urllib3>=1.26.0"],
-    setup_requires=["pytest-runner", "pylint-runner", "setuptools_scm", "wheel"],
     tests_require=["httpretty", "pytest", "pylint"],
-    extras_require={
-        "analytics": ["pandas>=1.2.4", "pyyaml>=5.4.1", "fastparquet>=0.6.3"],
-    },
+    extras_require={"analytics": ["pandas>=1.2.4", "pyyaml>=5.4.1", "fastparquet>=0.6.3"]},
     packages=PACKAGES,
     package_dir={pkg: os.path.join(*pkg.split(".")) for pkg in PACKAGES},
-    cmdclass={"pylint": PylintCmd, "mypy": MypyCmd},
+    cmdclass={"pylint": Pylint, "mypy": Mypy, "black": Black, "pytest": Pytest},
```

Comment:
Should we also have custom command for isort? Or maybe we could merge commands for black and isort into a single comand named (for example) `format`.

Reply:
I will add "format" command to do black and isort in a single shot